scripts/update-values-schema-json.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
+# Generate json schema for chart values.
+#
+# https://github.com/karuppiah7890/helm-schema-gen
```

Comment:
As you know, helm-schema-gen is already unmaintained and we don't know when it will be unavailable. It works as expected at this time, so we should have no problem using it, but we may want to fork the repository into scalar-labs org in case it is removed.

Reply:
Thank you :bow:
This will be handled separately.